Mathlib/LinearAlgebra/QuadraticForm/Real.lean

```diff
@@ -7,7 +7,6 @@ import Mathlib.LinearAlgebra.QuadraticForm.IsometryEquiv
 import Mathlib.Data.Sign
 import Mathlib.Algebra.CharP.Invertible
 import Mathlib.Analysis.RCLike.Basic
-import Mathlib.Data.Complex.Abs
```

Comment:
Presumably, this file pulled in just the instance above (when it could have been found through other means now). Same for the other file.